packages/tools/fuzzer/utils/randomString.ts

```diff
@@ -0,0 +1,21 @@
+type OnRandomInt = (low: number, high: number)=> number;
+
+const randomString = (randInt: OnRandomInt, length: number) => {
+	const charCodes = [];
+	for (let i = 0; i < length; i++) {
+		charCodes.push(randInt(0, 0xFFFF));
+	}
```

Comment:
Previously, folder titles, note bodies, and note titles were mostly pre-set strings (e.g. `Test... (<random number here>)`). With this change,
- Strings are now generated from a pseudorandom list of code points (see [String.fromCodePoint](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/fromCodePoint)).
- The pseudorandom number generator used here is the same one used elsewhere. As such, it's controlled by the `--seed` flag.
- There are a few limitations on the content of the string (see the `.replace`s later in the file). This prevents the "was the note saved correctly" assertion from failing, but it would make sense to see if the assertion can be adjusted in the future.